hybrid/sites/flatirons_site.py

```diff
@@ -1,7 +1,7 @@
 flatirons_site = {
-    "lat": 35.2018863,
-    "lon": -101.945027,
-    "elev": 1099,
+    "lat": 39.91,
+    "lon": -105.22,
+    "elev": 1800,
```

Comment:
Could you set the lat and lon to 39.7555 and -105.2211 so that it matches with the names of the solar and wind resources files in this folder: https://github.com/NREL/HOPP/tree/master/resource_files/solar

This way all code that uses the flatirons site will re-use those resource files because the numbers match up exactly.

Reply:
@dguittet, in response to the first comment about modifying locations.py: I agree, the Texas location is still of interest and it would be easiest to revert the tests to continue to use the TX location. I would like to reduce ambiguity in the file names if possible, can I propose an alternative solution? We can create a new hybrid/sites/amarillo_site.py that is a copy of the [original flatirons_site.py](https://github.com/NREL/HOPP/blob/master/hybrid/sites/flatirons_site.py), keep updates to flatirons_site.py in this PR (possibly use these more precise [coords and elev?](https://midcdmz.nrel.gov/apps/sitehome.pl?site=NWTC)),  and update the tests to reference amarillo_site.py instead of flatirons_site.py? I believe this would accomplish the same goal while being explicit in the sites we are testing.


In response to the flatirons_site.py comment above: 39.7555, -105.2211 appears to be a separate site in the heart of downtown Golden, CO and is a hardcoded value in test_flicker_mismatch.py, test_flicker.py, test_reopt.py, test_resource_download.py, test_solar_wind.py and their associated main HOPP files.

Based on what I'm seeing it appears the 39.7555, -105.2211 hardcoded values should be left as is and their associated data files left in the resource_files directory to maintain the original state of the tests.

That being said if you want lat/lon to a higher precision for flatirons we can get a more precise point like this one https://midcdmz.nrel.gov/apps/sitehome.pl?site=NWTC

Reply:
@dakotaramos For the examples, it's fine if they run the location in CO vs TX. I've updated many tests to use specific weather files: https://github.com/NREL/HOPP/blob/d49cbd48b9ffb2145cd1b29e2370136fd01b4c37/tests/hybrid/test_hybrid.py#L11

This will get rid of both this current issue about needing to update the test results given the lat / lon changes. In this case, I'm supportive of modifying flatirons_site.py to use the correct coords and elev. Do feel free to use more exact coordinates.

The site is more about the boundary dimensions / polygon than it is about lat / lon, so I don't see a problem with replacing the lat/lon. So flatirons looks like the actual site there, and the circular one is a circle, etc. So, I don't think we'll need the duplicate amarillo_site.py.